public/app/core/components/sidemenu/__snapshots__/DropDownChild.test.tsx.snap

```diff
@@ -3,6 +3,7 @@
 exports[`Render should render component 1`] = `
 <li
   className="divider css-1djiy5s"
+  data-testid="sidemenu-item-undefined"
```

Comment:
This is undefined because the first item of the menu is the search button and grafana doesn't add id to the search button. This is fine since we are not going to test grafana stuff in the menu, we will only check for the items that we added.